Google.Api.Generator.Tests/ProtoTests/Basic/Testing.Basic/Testing.Basic.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+
+    <!-- TODO: Version defaults to 1.0.0, edit as required -->
+    <Version>1.0.0</Version>
+
+    <!-- TODO: NuGet packaging options -->
+    <!--
+      <Description>
+        TODO: Description of this library.
+      </Description>
+      <PackageTags>TODO;Library;Tags</PackageTags>
+      <Copyright>TODO: Copyright Statement</Copyright>
+      <Authors>TODO:Authors</Authors>
+      *** TODO: These Icon, License, Project, and repo settings *MUST* be checked and edited ***
+      *** The values given are just examples ***
+      <PackageIconUrl>TODO: https://cloud.google.com/images/gcp-icon-64x64.png</PackageIconUrl>
+      <PackageLicenseUrl>TODO: https://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
+      <PackageProjectUrl>TODO: https://github.com/googleapis/google-cloud-dotnet</PackageProjectUrl>
+      <RepositoryType>TODO: git</RepositoryType>
+      <RepositoryUrl>TODO: https://github.com/googleapis/google-cloud-dotnet</RepositoryUrl>
+    -->
+
+    <!-- TODO: Configure package signing -->
+    <!--
+      <AssemblyOriginatorKeyFile>...</AssemblyOriginatorKeyFile>
+      <SignAssembly>true</SignAssembly>
+      <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    -->
+
+    <!-- These items should not require editing -->
+    <TargetFrameworks>netstandard1.5;net45</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.5</TargetFrameworks>
```

Comment:
Another condition we could trim :)

Reply:
Thought I'd got all of these! Apparently not.

Reply:
This is already trimmed. Not sure why it's not showing up here.